src/message/messageActionSheet.js

```diff
@@ -89,6 +90,10 @@ const editMessage = async ({ message, actions }: MessageAuthAndActions) => {
   actions.startEditMessage(message.id);
 };
 
+const deleteMessage = async ({ auth, message }: AuthAndMessageType) => {
+  patchMessage(auth, '(deleted)', message.id);
```

Comment:
Cool. @saketkumar95 please make sure that you are sending an empty string. Besides that, the PR is ready to merge and I'll do so after the change.